src/lib/serviceRoutines/getContextEntitiesByEntityIdAndType.cpp

```diff
@@ -41,8 +43,18 @@
 *
 * getContextEntitiesByEntityIdAndType - 
 *
-* Invoked by:
-*  GET /v1/registry/contextEntities/type/{ETYPE}/id/{EID}
+* GET /v1/registry/contextEntities/type/{ETYPE}/id/{EID}
+*
+* URI parameters:
+*   - entity::type=XXX     (must coincide with entity::type in URL)
+*   - !exist=entity::type  (if set - error -- entity::type cannot be empty)
+*   - exist=entity::type   (not supported - ok if present, ok if not present ...)
```

Comment:
We don't support attributeFormat in this case as it doesn't make sense for discoverContextAvailability (in that case attributes doesn't have a value, as in NGS10). Is my argument correct?